dbms/src/Flash/Coprocessor/tests/gtest_streaming_writer.cpp

```diff
@@ -177,39 +176,5 @@ try
 }
 CATCH
 
-TEST_F(TestStreamingWriter, testSendExecutionSummary)
```

Comment:
Why remove ut？

Reply:
Because in this pr, we wouldn't sending execution summary in StreamingWriter